src/CoreWCF.Http/src/CoreWCF/WSHttpBindingBase.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using CoreWCF.Channels;
+using CoreWCF.Runtime;
+
+using System.Xml;
+using CoreWCF.Configuration;
+
+namespace CoreWCF
+{
+   public abstract class  WSHttpBindingBase : Binding //, IBindingRuntimePreferences
+    {
+        private TextMessageEncodingBindingElement _textEncoding;
+
+        protected WSHttpBindingBase()
+            : base()
+        {
+            Initialize();
+        }
+
+        protected WSHttpBindingBase(bool reliableSessionEnabled) : this()
+        {
+            if (reliableSessionEnabled)
+            {
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        //[DefaultValue(HttpTransportDefaults.BypassProxyOnLocal)]
+        //public bool BypassProxyOnLocal
+        //{
+        //    get { return HttpTransport.BypassProxyOnLocal; }
+        //    set
+        //    {
+        //        HttpTransport.BypassProxyOnLocal = value;
+        //        HttpsTransport.BypassProxyOnLocal = value;
+        //    }
+        //}
+
+        [DefaultValue(false)]
+        public bool TransactionFlow
+        {
+            get { return false; }
+            set
+            {
+                if (value)
+                {
+                    throw new PlatformNotSupportedException();
+                }
+            }
+        }
+
+       // [DefaultValue(TransportDefaults.MaxBufferPoolSize)]
+        public long MaxBufferPoolSize
```

Comment:
Uncomment the default value. There's some potential external configuration helpers coming where this would be helpful.